packages/plugin-snowplow/src/androidMain/kotlin/ly.iterative.itly.snowplow/SnowplowPlugin.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * Android SnowplowPlugin
+ */
+package ly.iterative.itly.snowplow
+
+import ly.iterative.itly.*
+
+import com.snowplowanalytics.snowplow.Snowplow;
+import com.snowplowanalytics.snowplow.controller.TrackerController
+import com.snowplowanalytics.snowplow.event.SelfDescribing
+import com.snowplowanalytics.snowplow.payload.SelfDescribingJson
+
+actual class SnowplowPlugin actual constructor(
+    options: SnowplowOptions
+) : Plugin(ID) {
+    companion object {
+        @JvmField
+        val ID = "snowplow"
+    }
+
+    private var config: SnowplowOptions = options
+    private lateinit var logger: Logger
+    private var snowplow: TrackerController? = null
+
+    val client: TrackerController?
+        get() = this.snowplow
+
+    override fun load(options: PluginLoadOptions) {
+        logger = options.logger
+        logger.debug("[plugin-snowplow] load")
+
+        if (config.tracker != null) {
+            this.snowplow = config.tracker
+        }
+        else {
+            this.snowplow = Snowplow.getDefaultTracker()
+        }
+    }
+
+    override fun identify(userId: String?, properties: Properties?) {
+        logger.debug("[plugin-snowplow] identify(userId=$userId, properties=${properties?.properties})")
+        if (userId == null) {
```

Comment:
Why return earlier on undefined userId? As Android in single-tenant, I think undefined is valid for "anonymous" users.

Reply:
Must have been looking at a multitenant implementation somewhere in Segment plugin, will fix.